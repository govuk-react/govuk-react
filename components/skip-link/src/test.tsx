import React from 'react';
import { mount } from 'enzyme';

import { SkipLinkDocumented as SkipLink } from '.';

describe('SkipLink', () => {
  it('matches snapshot', () => {
    expect(mount(<SkipLink>Example</SkipLink>)).toMatchSnapshot('SkipLink');
  });
});
