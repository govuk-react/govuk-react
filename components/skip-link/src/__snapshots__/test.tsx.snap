// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkipLink matches snapshot: SkipLink 1`] = `
.c0 {
  position: absolute !important;
  width: 1px !important;
  height: 1px !important;
  margin: 0 !important;
  overflow: hidden !important;
  -webkit-clip: rect(0 0 0 0) !important;
  clip: rect(0 0 0 0) !important;
  -webkit-clip-path: inset(50%) !important;
  clip-path: inset(50%) !important;
  border: 0 !important;
  white-space: nowrap !important;
  font-family: "nta",Arial,sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-size: 14px;
  line-height: 1.1428571428571428;
  display: block;
  padding: 10px 15px;
}

.c0:active,
.c0:focus {
  position: static !important;
  width: auto !important;
  height: auto !important;
  margin: inherit !important;
  overflow: visible !important;
  -webkit-clip: auto !important;
  clip: auto !important;
  -webkit-clip-path: none !important;
  clip-path: none !important;
  white-space: inherit !important;
}

.c0:focus {
  outline: 3px solid #ffdd00;
  outline-offset: 0;
  background-color: #ffdd00;
}

.c0:link,
.c0:visited,
.c0:hover,
.c0:active,
.c0:focus {
  color: #0b0c0c;
}

@media print {
  .c0 {
    font-family: sans-serif;
  }
}

@media print {
  .c0:link,
  .c0:visited,
  .c0:hover,
  .c0:active,
  .c0:focus {
    color: #000;
  }
}

@media only screen and (min-width:641px) {
  .c0 {
    font-size: 16px;
    line-height: 1.25;
  }
}

@media print {
  .c0 {
    font-size: 14px;
    line-height: 1.2;
  }
}

@supports (padding:max(calc(0px))) {
  .c0 {
    padding-right: max(15px,calc(15px + env(safe-area-inset-right)));
    padding-left: max(15px,calc(15px + env(safe-area-inset-left)));
  }
}

<SkipLinkDocumented
  href="#content"
>
  <src__SkipLink
    href="#content"
  >
    <a
      className="c0"
      href="#content"
    >
      Example
    </a>
  </src__SkipLink>
</SkipLinkDocumented>
`;
